samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/teamnotification.jsx

```diff
@@ -0,0 +1,99 @@
+// <copyright file="dashboard.jsx" company="Microsoft Corporation">
```

Comment:
File name is different and here it's different.
Rename it as team-notification.jsx